src/Sylius/Bundle/AdminBundle/Action/ResendOrderConfirmationEmailAction.php

```diff
@@ -45,6 +46,17 @@ public function __construct(
                 RequestStack::class,
             );
         }
+
+        if ($this->orderEmailManager instanceof OrderEmailManagerInterface) {
+            trigger_deprecation(
+                'sylius/admin-bundle',
+                '1.13',
+                'Passing an instance of %s as constructor argument for %s is deprecated and will be removed in Sylius 2.0. Pass an instance of %s instead.',
```

Comment:
I noticed that you suggested changing the name of the parameter. Maybe we should add information here stating that in version 2.0, the name of `orderEmailManager` will be changed to `resendOrderConfirmationEmailDispatcher`. WDYT?